GOALS.md

```diff
@@ -0,0 +1,16 @@
+# Presto Foundation Strategic Goals 
+
+The Presto Foundation has published its goals for 2020:
+
+https://docs.google.com/presentation/d/1Lg9sQiby2r7ISlFbe6tWhlqLOGOcoknsaF41Rn5Cmks/edit#slide=id.g75dc891a4b_0_56
+
+The goals will be routinely revisited and updated for each year with input from the membership and community.
+
+## 2020
+
+* Unify Presto under one neutral and openly governed community
+* Cultivate adoption of Presto
+* Create value for all PF developers and members
```

Comment:
This is the first time PF abbreviation is used in this doc. It would be better to use the full form when the abbreviation is first used so that readers know what PF is. So, you can either update Line 3 above and use `Presto Foundation (PF)` or you can use the full form + abbreviation here.